LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2021, 2022 Simon Lydell
```

Comment:
@lydell I'm following this [guide](https://gist.github.com/fbaierl/1d740a7925a6e0e608824eb27a429370), hope it is fine!

Reply:
I’m not an expert myself, but looks good!